src/main/java/com/robosh/ejournal/config/BeanConfig.java

```diff
@@ -10,9 +12,19 @@
 public class BeanConfig {
 
     @Bean
-    public ModelMapper getModelMapper(){
+    public ModelMapper getModelMapper() {
         ModelMapper modelMapper = new ModelMapper();
         modelMapper.getConfiguration().setSkipNullEnabled(true).setMatchingStrategy(MatchingStrategies.STRICT);
         return modelMapper;
     }
+
+    @Bean
+    public BeansFactory getBeansFactory(
+            ValidationService validationService
+    ) {
```

Comment:
why new line

Reply:
Because it can be extended and include a lot of classes, but ok, for now, I will change it